react/components/admin/pages/Form/ConditionalTemplatePicker.tsx

```diff
@@ -12,7 +12,14 @@ import SelectConditions from './SelectConditions'
 
 export interface ConditionalTemplatePickerProps {
   condition: ConditionFormsData
-  formErrors: Partial<{ [key in keyof Route]: string }>
+  formErrors: Omit<Partial<{ [key in keyof Route]: key }>, 'pages'> & {
```

Comment:
can't you define this type somewhere instead of rewriting it 3 times?

Reply:
Done :)

Just me being lazy.